docs/source/operations/overview.rst

```diff
@@ -19,10 +19,27 @@ The charts are available in the Substra repository:
 
     helm repo add substra https://substra.github.io/charts
 
-For each component section below, configuration options relates to the component's chart unless specified otherwise.
+For each component in the section below, configuration options relates to the component's chart unless specified otherwise.
```

Comment:
```suggestion
For each component in the section below, configuration options relate to the component's chart unless specified otherwise.
```

Options is plural, so no s to the verb